test/quantization/test_quantize_fx.py

```diff
@@ -1486,6 +1487,39 @@ def forward(self, x):
             expected_node_occurrence=count_check,
             expected_node_list=order_check)
 
+    def test_float_functional(self):
+        class M(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.ff1 = nnq.FloatFunctional()
+                self.ff2 = nnq.FloatFunctional()
+                self.ff3 = nnq.FloatFunctional()
+
+            def forward(self, x):
+                x = self.ff1.add(x, x)
+                x = self.ff2.mul(x, x)
+                x = self.ff3.add_relu(x, x)
+                return x
+
+        m = M()
+        m.eval()
+        qconfig_dict = {"": default_qconfig}
+        m = prepare_fx(m, qconfig_dict)
+        node_occurrence = {
+            ns.call_module(torch.quantization.MinMaxObserver): 4,
+            ns.call_module(torch.nn.quantized.FloatFunctional): 0
+        }
+        self.checkGraphModuleNodes(m, expected_node_occurrence=node_occurrence)
+        node_list = [
+            ns.call_function(torch.quantize_per_tensor),
+            ns.call_function(torch.ops.quantized.add),
+            ns.call_function(torch.ops.quantized.mul),
+            ns.call_function(torch.ops.quantized.add_relu),
+            ns.call_method('dequantize')
+        ]
+        m = convert_fx(m)
+        self.checkGraphModuleNodes(m, expected_node_list=node_list)
```

Comment:
makes sense, should we also add a numerical equivalency test for these for eager vs FX graph?

Reply:
sure, I can add a numerical test